test-data/run.test

```diff
@@ -692,7 +692,7 @@ assert f(1) == 2
 def f() -> str:
     return 'some string'
 def g() -> str:
-    return 'some\a \v \t \x7f " \n \0string 🐍'
+    return 'some\a \v \t \x7f " \n \0string \U0001F40D'
```

Comment:
With the latest mypy master UTF-8 may work here.

Reply:
:+1: